phylogenetic/rules/merge_sequences_usvi.smk

```diff
@@ -21,22 +21,141 @@ This part of the workflow usually includes the following steps:
 
 """
 
-rule append_usvi:
-    """Appending USVI sequences"""
-    input:
-        sequences = "data/sequences.fasta",
-        metadata = "data/metadata.tsv",
-        usvi_sequences = "data/sequences_usvi.fasta",
-        usvi_metadata = "data/metadata_usvi.tsv"
+# ------------- helper functions to collect, merge & download input files ------------------- #
+NEXTSTRAIN_PUBLIC_BUCKET = "s3://nextstrain-data/"
+
+def _parse_config_input(input):
+    """
+    Parses information from an individual config-defined input, i.e. an element within `config.inputs` or `config.additional_inputs`
+    and returns information snakemake rules can use to obtain the underlying data.
+
+    The structure of `input` is a dictionary with keys:
+    - name:string (required)
+    - metadata:string (optional) - a s3 URI or a local file path
+    - sequences:string|dict[string,string] (optional) - either a s3 URI or a local file path, in which case
+      it must include a '{segment}' wildcard substring, or a dict of segment → s3 URI or local file path,
+      in which case it must not include the wildcard substring.
```

Comment:
This signature should be adjusted to reflect that we (zika!) no longer support the individual-segment syntax

Reply:
My [attempt](https://github.com/nextstrain/zika/pull/80/commits/9e8e44b8ef7bee04b6dd610ec68059650ecfabb7), please feel free to reword